### PR TITLE
[new release] colombe, sendmail and sendmail-lwt (0.4.2)

### PR DESCRIPTION
**Pull request by dinosaure**
SMTP protocol in OCaml

- Project page: <a href="https://github.com/mirage/colombe">https://github.com/mirage/colombe</a>
- Documentation: <a href="https://mirage.github.io/colombe/">https://mirage.github.io/colombe/</a>

##### CHANGES:

- Add `Path.of_string_exn` (@dinosaure, mirage/colombe#40)
- Be resilient about 334 argument and add regression test (@dinosaure, @jsthomas, mirage/colombe#41)

### CODE REVIEW

**Comment by avsm** [comment]
```
2021-07-27 09:27.58: Error in sendmail-lwt.0.4.2: Your dune-project file indicates that this package requires at least dune 2.0 but your opam file only requires dune >= 1.8. Please check which requirement is the right one, and fix the other.
2021-07-27 09:27.58: Error in sendmail.0.4.2: Your dune-project file indicates that this package requires at least dune 2.0 but your opam file only requires dune >= 1.8. Please check which requirement is the right one, and fix the other.
2021-07-27 09:27.58: Error in colombe.0.4.2: Your dune-project file indicates that this package requires at least dune 2.0 but your opam file only requires dune >= 1.8.0. Please check which requirement is the right one, and fix the other.
```

File: packages/colombe/colombe.0.4.2/opam [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+description: """SMTP protocol according RFC5321 without extension."""
+
+build: [
+  [ "dune" "build" "-p" name "-j" jobs ]
+  [ "dune" "runtest" "-p" name "-j" jobs ] {with-test}
+]
+
+depends: [
+  "ocaml" {>= "4.03.0"}
+  "dune" {>= "1.8.0"}
```
Comment (avsm) [inline packages/colombe/colombe.0.4.2/opam:19]:
```suggestion
  "dune" {>= "2.0.0"}
```

File: packages/sendmail-lwt/sendmail-lwt.0.4.2/opam [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+description: """A library to be able to send an email with LWT and TLS."""
+
+build: [
+  [ "dune" "build" "-p" name "-j" jobs ]
+  [ "dune" "runtest" "-p" name "-j" jobs ] {with-test}
+]
+
+depends: [
+  "ocaml" {>= "4.03.0"}
+  "dune" {>= "1.8"}
```
Comment (avsm) [inline packages/sendmail-lwt/sendmail-lwt.0.4.2/opam:19]:
```suggestion
  "dune" {>= "2.0.0"}
```

File: packages/sendmail/sendmail.0.4.2/opam [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+description: """A library to be able to send an email."""
+
+build: [
+  [ "dune" "build" "-p" name "-j" jobs ]
+  [ "dune" "runtest" "-p" name "-j" jobs ] {with-test}
+]
+
+depends: [
+  "ocaml" {>= "4.03.0"}
+  "dune" {>= "1.8"}
```
Comment (avsm) [inline packages/sendmail/sendmail.0.4.2/opam:19]:
```suggestion
  "dune" {>= "2.0.0"}
```

**Comment by kit-ty-kate** [comment]
Thanks! Could you return the fixes in the opam file upstream?

**Pull request merged**